docs/SQLiteReport.rst

```diff
@@ -0,0 +1,261 @@
+Working with SQLite report
+==========================
+
+From the very beginning, we didn't want to impose our vision on treating the results of mutation testing. Some people do not care about the mutation score, while others do care, but want to calculate it slightly differently.
```

Comment:
Looking through the whole diff: a width limit is used sometimes and sometimes is not. My preference for RST would still remain 80, but feel free to use your best judgment.

Reply:
I put the doc into Grammarly before publishing and it forced me to remove all the new-lines, I should've put them back though.

I'll do another round of cleanups in a separate PR and I'll go over all the other documents as well.